kurtosis-module/impl/module_io/participant_log_level/participan_log_level.go

```diff
@@ -0,0 +1,18 @@
+package participant_log_level
```

Comment:
I have to move these constants here to avoid an import cycle error that happened when they were in the original place